salt/modules/textfsm_mod.py

```diff
@@ -0,0 +1,457 @@
+# -*- coding: utf-8 -*-
+'''
+TextFSM
+=======
+
+.. versionadded:: Oxygen
+
+Execution module that processes plain text and extracts data
+using TextFSM templates. The output is presented in JSON serializable
+data, and can be easily re-used in other modules, or directly
+inside the renderer (Jinja, Mako, Genshi, etc.).
+
+:depends:   - textfsm Python library
+
+.. note::
+
+    For Python 2/3 compatibility, it is more recommended to
+    install the ``jtextfsm`` library: ``pip install jtextfsm``.
+'''
+from __future__ import absolute_import
+
+# Import python libs
+import os
+import logging
+
+# Import third party modules
+try:
+    import textfsm
+    HAS_TEXTFSM = True
+except ImportError:
+    HAS_TEXTFSM = False
+
+try:
+    import clitable
+    HAS_CLITABLE = True
+except ImportError:
+    HAS_CLITABLE = False
+
+# Import salt modules
+import salt.utils.files
```

Comment:
Hi @cachedout yes, I saw your suggestion - thanks! I'll make this change too, it's in my long queue, but given that's a new module of Oxygen has a lower prio.
Don't worry, I will let you know when ready. Cheers!